recipes/newrelic/apm/php/debian.yml

```diff
@@ -40,6 +40,7 @@ preInstall:
       IS_PHP_INSTALLED=$(which php | wc -l)
       if [ "$IS_PHP_INSTALLED" -eq 0 ]; then
         # PHP not installed, no detection
+        echo "Error: PHP is not installed. Please install PHP, configure and enable your web server (Apache or NGINX), then retry the installation." >&2
```

Comment:
@sanderblue are those messages displayed to the user, since it's in the "requireAtDiscovery" detection step?

Reply:
No, I guess they aren't since it's in this early step, but they do show up when using `--debug`.